src/js/Controllers/TTSController.js

```diff
@@ -182,26 +145,19 @@ class TTSController {
             case "SetGlobalProperties":
                 return true
             case "Speak":
-                if (this.speakID) {
+                if (this.currentlyPlaying) {
                     return { 
                         rpc: RpcFactory.ErrorResponse(rpc, 4, "Speak request already in progress")
                     };
                 }
                 var ttsChunks = rpc.params.ttsChunks
-                this.filePlaylist = []
                 for (var i=0; i<ttsChunks.length; i++) {
                         this.filePlaylist.push(ttsChunks[i])
                 }
-                this.speakID = rpc.id;
-                this.listener.send(RpcFactory.TTSStartedNotification());
+                this.filePlaylist.push({ type: 'REPLY', id: rpc.id });
```

Comment:
Please add code comments for this reply type since a developer might assume this is a type associated with the hmi api

Reply:
good point, I added a comment in two places in https://github.com/smartdevicelink/generic_hmi/pull/406/commits/4c4152ada070e01fa0501a5b95929969699b115f